### PR TITLE
edit app before approval

### PR DESCRIPTION
**Pull request by Puiu**
edit app before approval

### CODE REVIEW

File: config/my.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
     "IsDebug" : true,
-    "jwtSecret" : "",
```
Comment (dmfarcas) [inline config/my.json]: pus fisierul din greseala?

File: api/server.js [hunk 2]
```diff
@@ -255,6 +255,18 @@ router.route('/updateapp/:appid')
 
     });
 
+
+router.route('/editapp')
+    .put(
+    function (req, res) {
+        //console.log(req.body);
```
Comment (dmfarcas) [inline api/server.js]: uitat log

File: api/response-formatter.js [hunk 3]
```diff
@@ -41,6 +43,23 @@ ResponseFormatter.prototype = {
         }
         }
 
+    },
+    FormatFromResult(res, result){
+            
+            result = result.toLowerCase()
```
Comment (dmfarcas) [inline api/response-formatter.js]: e necesara reasignarea?

File: api/server.js (hunk 2 above)
Comment (lrdTnc) [reply api/server.js]: removed it

File: api/response-formatter.js (hunk 3 above)
Comment (lrdTnc) [reply api/response-formatter.js]: removed it

File: config/my.json (hunk 1 above)
Comment (lrdTnc) [reply config/my.json]: changes reverted

**Pull request merged**